enable/base_tool.py

```diff
@@ -170,5 +170,5 @@ def _deactivate(self):
     def deactivate(self, component=None):
         """ Handles this component no longer being the active tool.
         """
-        # Compatibility with new AbstractController interface
+        # Compatibility with [Chaco's] AbstractController interface
```

Comment:
Square brackets around `[Chaco's]`?  A search and replace gone wrong?
```suggestion
        # Compatibility with Chaco's AbstractController interface
```

Occurs a couple of other places too.

Reply:
No, that was intentional.